src/MotoPlusExterns.h

```diff
@@ -15,4 +15,11 @@ extern size_t mpNumBytesFree(void);
 
 extern MP_GRP_ID_TYPE mpCtrlGrpNo2GrpId(int grp_no);
 
+//M+ only defines MAX_ALARM_COUNT, but we'd like to avoid magic nrs
+//as much as possible, so we define our own here for errors.
+//But only if something hasn't been defined already.
+#ifndef MAX_ERROR_COUNT
+#define MAX_ERROR_COUNT 1
+#endif
```

Comment:
we could perhaps also put this in the platform lib, but seeing as it's basically something that should be part of M+, I decided to treat it like something that *is* part of M+, but just not `define`d in the (default) headers.